es-ds-components/components/es-card.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <component
+        :is="defaultTag =='nuxt-link' ? NuxtLink: defaultTag"
+        class="es-card"
+        :class="actualVariant"
+        :to="to ? to : href"
+        v-bind="$attrs">
+        <slot />
+    </component>
+</template>
+
+<script setup lang="ts">
+// specific workaround for getting a reference to an auto-imported but not globally registered
+// component like NuxtLink, so that it can be used in a <component :is=""> context.
+// this resolves to an import statement, according to:
+// https://github.com/nuxt/nuxt/issues/13659#issuecomment-1573568006
+const NuxtLink = resolveComponent('NuxtLink');
+
+const props = defineProps({
+    href: {
+        type: String,
+        default: undefined
+    },
+    tag: {
+        type: String,
+        default: 'div'
+    },
+    to: {
+        type: String,
+        default: undefined
+    },
+    variant: {
+        type: String,
+        default: 'display',
+        required: false
+    }
+});
+
+const actualVariant = computed(() => {
+    // if href is specified, force variant to be interactive
+    if (props.href) {
+                return 'interactive';
+            }
```

Comment:
It's odd to me that these are computed properties rather than just plain old variables.. it would only make a difference if we expected them to change while the page was already rendered. But it's the same in the old design system so I'm inclined to leave this as is. One small ask though- can you fix the indentation? We should have a linter and formatter in place pretty soon hopefully.